code/src/components/App/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import update from 'immutability-helper';
+
+import Thought from 'components/Thought';
+import Form from 'components/Form';
+import Main from 'components/Styled/Main';
+
+import { URL } from 'helpers/reusables';
+
+const App = () => {
+  const [thoughts, setThoughts] = useState([]);
+
+  const fetchThoughts = () => {
+    fetch(URL)
+      .then((res) => res.json())
+      .then((data) => {
+        setThoughts(data);
+      })
+      .catch((err) => console.log(err));
```

Comment:
Agreed, I had thought about error handling, but since it was sort of out of scope for these projects I decided to focus on other stuff. But maybe it's better to not have that catch() at all if I have no handler for it... 🤔

Reply:
console.log is never welcome in the production. You need to remove it 🔫